olm_deploy/scripts/catalog-deploy.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/sh 
 set -eou pipefail
 
-export IMAGE_ELASTICSEARCH_OPERATOR=${IMAGE_ELASTICSEARCH_OPERATOR:-quay.io/openshift/origin-elasticsearch-operator:latest}
-export IMAGE_ELASTICSEARCH6=${IMAGE_ELASTICSEARCH6:-quay.io/openshift/origin-logging-elasticsearch6:latest}
-export IMAGE_LOGGING_KIBANA6=${IMAGE_LOGGING_KIBANA6:-quay.io/openshift/origin-logging-kibana6:latest}
-export IMAGE_OAUTH_PROXY=${IMAGE_OAUTH_PROXY:-quay.io/openshift/origin-elasticsearch-proxy:latest}
+export IMAGE_ELASTICSEARCH_OPERATOR=${IMAGE_ELASTICSEARCH_OPERATOR:-registry.svc.ci.openshift.org/ocp/4.5:elasticsearch-operator}
```

Comment:
is it possible to use an env var for the image name version? (`4.5`) to make it easier for updating in our future releases?

Reply:
in general i expect this var to be populated by the ci job, or explicitly set by the developer, but yeah i can make the version an env var for the defaulting case.

Reply:
updated